service_container.rst

```diff
@@ -612,13 +618,18 @@ But, you can control this and pass in a different logger:
     .. code-block:: php
 
         // config/services.php
+        namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
         use App\Service\MessageGenerator;
-        use Symfony\Component\DependencyInjection\Reference;
 
-        $container->autowire(MessageGenerator::class)
-            ->setAutoconfigured(true)
-            ->setPublic(false)
-            ->setArgument('$logger', new Reference('monolog.logger.request'));
+        return function(ContainerConfigurator $configurator) {
+            $container = $configurator->services();
+            $container->set(SiteUpdateManager::class)
+                ->autowire()
```

Comment:
Can you please add the same comments as other examples and remove needless boilerplate?

Reply:
Y, definitely, not sure why I did it like that.